### PR TITLE
Replace erroneous StratisCliNoChangeError

### PR DESCRIPTION
**Pull request by waltdisgrace**
Partially addresses #310.
Related: https://github.com/stratis-storage/project/issues/51.

### CODE REVIEW

**Comment by waltdisgrace** [comment]
rebased.

File: src/stratis_cli/_actions/_top.py [hunk 1]
```diff
@@ -199,7 +199,14 @@ def destroy_pool(namespace):
             raise StratisCliEngineError(rc, message)
 
         if not changed:
-            raise StratisCliNoChangeError("destroy", namespace.pool_name)
+            raise StratisCliIncoherenceError(
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_top.py]: Go ahead and put the ```pragma: no cover``` annotation on the ```if not changed``` line, as we do with the other ```Incoherence``` errors.

**Comment by mulkieran** [comment]
@waltdisgrace Please rebase one last time.

**Comment by waltdisgrace** [comment]
rebased.

**Comment by mulkieran** [comment]
Travis tests have passed, GitHub just isn't picking up the change.

**Pull request merged**